model-view-viewmodel/pom.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+    <artifactId>java-design-patterns</artifactId>
+    <groupId>com.iluwatar</groupId>
+    <version>1.24.0-SNAPSHOT</version>
+  </parent>
+	<groupId>com.iluwatar</groupId>
+	<artifactId>model-view-viewmodel</artifactId>
+	<version>1.24.0-SNAPSHOT</version>
+	<properties>
+		<zk.version>9.0.0</zk.version>
+		<commons-io>2.6</commons-io>
```

Comment:
In general, all the version numbers and at least the dependencies that are used elsewhere too should be placed in the root `pom.xml`.

Reply:
We should use the same versions of dependencies in all the patterns.

Reply:
done